tensorflow/tools/ci_build/builds/test_on_install.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
I don't thing we need this file. It should go to gpu_pip.sh. There is no reason to have one file to build pip and other to build and test. We always want to test it.